src/ResourceManager/KeyVault/Commands.KeyVault.Test/Scripts/PSHCommon/Common.ps1

```diff
@@ -298,7 +298,7 @@ function Wait-Seconds
 {
     param([int] $timeout)
     
-    [Microsoft.WindowsAzure.Testing.TestUtilities]::Wait($timeout * 1000)
+    Sleep -s $timeout
```

Comment:
These tests are run manually and will not import the standard azure powershell test functions. This function is only called when we are running the tests manually.